src/metadata_tool/sources/confluence.clj

```diff
@@ -51,14 +57,24 @@
    (str/split url #"/") 4))
 
 (defn content
-  [id]
-  (:value (:storage (:body (:body
-                            (cget "content/" id "?expand=body.storage"))))))
+  [path]
+  (try
+    (let [driver (selenium/init-driver)
+          wdw (WebDriverWait. driver selenium-timeout)
+          condition (ExpectedConditions/elementToBeClickable (By/id "main-content"))
+          host (:host (:confluence cfg/config))
+          url (str host "/wiki" path)]
+      (.get driver url)
+      (.until wdw condition)
+      (-> driver
+        (.findElement (By/xpath "//*[self::h1 or self::h2 and text()='Attendees']/following::table"))
+        (.getAttribute "innerHTML")))
+  ; TODO - return nil AND print out error message in console
+  (catch Exception e nil)))
```

Comment:
maybe a log statement here?  Would help debugging.

Reply:
Thing is, I'm not sure about the code here; shall I replace `nil` with `(do (println "An Exception occurred blah") nil)`, or simply `(println "An Exception occurred blah")` ?

The fact that the `content` function is called from a `map` function sometimes changes the behaviour of the application, leading to an empty result (at least, this is what I've experienced in the past)